### PR TITLE
Typos and links in Chalet time policy page

### PR DESCRIPTION
**Pull request by owenblacker**

### CODE REVIEW

**Comment by samuele-mattiuzzo** [comment]
Looks good to me 👍 🚀

**Review by tomvaughan77 — APPROVED** [review]

File: guides/chalet_time_policy.md [hunk 1]
```diff
@@ -102,11 +102,11 @@ Scheduling team will giving the Talent team coordinators access to the Kimble re
 
 #### 4. Research and development
 
-The R&D team will maintain a visible backlog of tasks people with chalet time can do either to help build products or research new opportunities. This will be posted in the #chalettimeteam Slack channel at the beginning of each week. 
+The R&D team will maintain a visible backlog of tasks people with chalet time can do either to help build products or research new opportunities. This will be posted in the [`#chalet-time-team`](https://madetechteam.slack.com/archives/C03F23K2RL0) Slack channel at the beginning of each week.
```
Comment (adrianclay) [inline guides/chalet_time_policy.md:105]: Potentially useful information, markdown has a concept of [Reference style links](https://www.markdownguide.org/basic-syntax/#reference-style-links) which would reduce the number of places this URL was repeated.

**Review by adrianclay — APPROVED** [review]

**Comment by lukemorton** [comment]
Hey 👋  Thanks for contributing a commit!

If the changes have been approved and the build is passing, please feel free to merge! This PR will eventually be closed if it isn't merged.

**Comment by owenblacker** [comment]
> Hey 👋 Thanks for contributing a commit!
> 
> If the changes have been approved and the build is passing, please feel free to merge! This PR will eventually be closed if it isn't merged.

@lukemorton: I don't have merge access, I don'think. Do I need adding to a Made Tech team or sth?

**Pull request merged**